test/smoke_test/smoke_test.py

```diff
@@ -5,7 +5,6 @@
 
 
 def base_smoke_test():
-    import torchaudio  # noqa: F401
```

Comment:
Please do not change this. This script is for OSS testing, and this line is checking if the extension is properly configured.